src/coreclr/jit/llvm.cpp

```diff
@@ -319,6 +356,7 @@ Type* getLLVMTypeForVarType(var_types type)
     switch (type)
     {
         case var_types::TYP_BOOL:
+            return Type::getInt1Ty(_llvmContext);
```

Comment:
Is this change right? I thought you changed bool to map to Int8 in previous PR.

Reply:
Thanks for spotting, looks wrong and a merge error on my part.

Reply:
put back to i8